cmake/Modules/Findgrpc.cmake

```diff
@@ -26,7 +26,7 @@ find_package_handle_standard_args(grpc DEFAULT_MSG
     )
 
 set(URL https://github.com/grpc/grpc)
-set(VERSION bfcbad3b86c7912968dc8e64f2121c920dad4dfb)
+set(VERSION v1.10.1)
```

Comment:
Hashes guarantee that commit tagged by the version is not changed. Please use hashes for consistency with other dependencies.